tools/CMakeLists.txt

```diff
@@ -4,9 +4,18 @@ function(_pico_init_pioasm)
     find_package(Pioasm REQUIRED)
 endfunction()
 
+# PICO_DEFAULT_PIOASM_OUTPUT_FORMAT, default output format used by pioasm when using pico_generate_pio_header, group=build
```

Comment:
Could you please add this to the end of the line too?
```
, default=c-sdk
```

Reply:
i'm going to merge it now; @lurch you can add a PR to update that!

Reply:
oh, found handy new edit feature in github!